en/asgardeo/docs/apis/restapis/offline-user-onboard.yaml

```diff
@@ -0,0 +1,104 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title:  Offline User Onboard Management API
+  description: >-
+    This document specifies an **Offline user onboard management RESTFull API** for
+    Asgardeo.
+
+paths:
+  '/offline-invite-link':
+    post:
+      tags:
+        - Temporary Link
+      summary: Generates a random link that can be used to set a new password
+      description: |
+        'Generates a random and secured one time link that can be used to set a new password for a user onboarded to the organization'
+        
+        <b>Scope(Permission) required:</b> `internal_offline_invite`
+      operationId: generateLink
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InvitationRequest'
+        description: This represents the invitation request.
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            text/plain:
+              schema:
+                type: string
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl --location 'https://api.asgardeo.io/o/{organization-name}/api/users/v1/offline-invite-link' \
+            -H 'Content-Type: application/json' \
+            -H 'Accept: text/plain' \
+            -H 'Authorization: Bearer {bearer_token}' \
+            -d '{
+                "username": "johnDoe",
+                "userstore": "PRIMARY"
```

Comment:
Shouldn't this need to be DEFAULT?
Check other places as well

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/e53861162efdcd9c89b63d459cc797db303432cf